src/logic/export/TagLabelsExport.ts

```diff
@@ -0,0 +1,53 @@
+import {ExportFormatType} from "../../data/enums/ExportFormatType";
+import {LabelsSelector} from "../../store/selectors/LabelsSelector";
+import {ImageData, LabelName} from "../../store/labels/types";
+import {saveAs} from "file-saver";
+import {ExporterUtil} from "../../utils/ExporterUtil";
+import {ImageRepository} from "../imageRepository/ImageRepository";
+import {findLast} from "lodash";
+
+export class TagLabelsExporter {
+    public static export(exportFormatType: ExportFormatType): void {
+        switch (exportFormatType) {
+            case ExportFormatType.CSV:
+                TagLabelsExporter.exportAsCSV();
+                break;
+            default:
+                return;
+        }
+    }
+
+    private static exportAsCSV(): void {
```

Comment:
mam wrażenie że te kawałki kodu Ci się powtarzają. w ostatnim CR też to widziałem (albo bardzo podobne).
może jakieś dziedziczenie albo Util zrobić?